.github/workflows/pr-rebase-needed.yml

```diff
@@ -0,0 +1,20 @@
+name: "PR Needs Rebase"
+on:
+  # So that PRs touching the same files as the push are updated
+  push:
+  # So that the `dirtyLabel` is removed if conflicts are resolve
```

Comment:
This looks like it's copied from an example and doesn't really work as an explanation for BCD contributors. A comment saying when the workflow is triggered would be better.